packages/react/src/button/button.styles.ts

```diff
@@ -180,8 +180,8 @@ export const StyledButton = styled(
       },
       disabled: {
         true: {
-          bg: '$accents2',
-          color: '$accents4',
+          bg: '$accents2 !important',
```

Comment:
hey @Dhananjay-JSR please let's try to avoid using `!import` since it limits the customization, instead we can use compound variants to guarantee the styles as follows

```
  // color=success && disabled
      {
        color: 'success',
        disabled: true,
        css: {
          bg: '$accents2',
          color: '$accents4'
        }
  },
  ... do the same with the rest of colors
```

On the other hand, we have to also create variants for "bordered", "gradient", "flat", "light" and "ghost" alongside "disabled" 🙏🏻